index.html

```diff
@@ -6009,27 +6009,30 @@ <h2 id="AudioWorklet-Examples">
   }
 
   process (inputs, outputs, parameters) {
-    // Note that the input will be downmixed to mono; however, if no inputs are
+    // Note that the input will be down-mixed to mono; however, if no inputs are
     // connected then zero channels will be passed in.
-    let buffer = inputs[0][0];
-    let bufferLength = buffer.length;
-    let sum = 0, x = 0, rms = 0;
-
-    // Calculated the squared-sum.
-    for (let i = 0; i &lt; bufferLength; ++i) {
-      x = buffer[i];
-      sum += x * x;
-    }
+    if (inputs.length &gt; 0) {
```

Comment:
Actually, I think it needs to be `inputs[0][0].length` - we need to check the number of elements in the first channel of the first input.

WDYT?